### ISSUE TITLE
name in Metadata is not stable

### ISSUE THREAD

**Issue by Xuanwo** [Xuanwo/hdrs#52]
Sometimes, hdfs could return name like `file:/xxxx/xxx`

```hsell
Error: object error: (op: list, path: hdfs://127.0.0.1:9000/4f607808-c05f-44c1-b247-423c51d9bcc1/b91d079e-17c0-4d84-b317-2cc1466514ec, source: path doesn't have specified prefix)
```

**Comment by Xuanwo** [comment]
We need to parse this name and get correct `path`, `name`, `parent` and so on.

**Comment by Xuanwo** [comment]
Fixed by #53
